modules/exoplanets/exoplanetshelper.cpp

```diff
@@ -57,19 +58,37 @@ bool hasSufficientData(const ExoplanetDataEntry& p) {
 }
 
 glm::vec3 computeStarColor(float bv) {
-    std::ifstream colorMap(absPath(BvColormapPath), std::ios::in);
+    const ExoplanetsModule* module = global::moduleEngine->module<ExoplanetsModule>();
+    const std::string bvColormapPath = module->bvColormapPath();
+
+    std::ifstream colorMap(absPath(bvColormapPath), std::ios::in);
 
     if (!colorMap.good()) {
         LERROR(fmt::format(
             "Failed to open colormap data file: '{}'",
-            absPath(BvColormapPath)
+            absPath(bvColormapPath)
         ));
-        return glm::vec3(0.f, 0.f, 0.f);
+        return glm::vec3(0.f);
     }
 
-    const int t = static_cast<int>(round(((bv + 0.4) / (2.0 + 0.4)) * 255));
+    // Interpret the colormap cmap file
+    std::string line;
+    while (std::getline(colorMap, line)) {
+        if ((line[0] == '#') || line.empty()) {
```

Comment:
You should check the other way around;  otherwise if the line is empty, the `line[0]` will lead to a crash